README.md

```diff
@@ -72,19 +69,117 @@ require('calendar')
 ```
 app/javascript/calendar/index.js // gets loaded by require('calendar')
 app/javascript/calendar/components/grid.jsx
+app/javascript/calendar/styles/grid.sass
 app/javascript/calendar/models/month.js
 ```
 
-But it could also look a million other ways. The only convention that Webpacker enforces is the
-one where entry points are automatically configured by the files in `app/javascript/packs`.
+```erb
+<%# app/views/layout/application.html.erb %>
+<%= javascript_pack_tag 'calendar' %>
+<%= stylesheet_pack_tag 'calendar' %>
+```
+
+But it could also look a million other ways.
+
+## Advanced Configuration
+
+By default, webpacker offers simple conventions for where the webpack configs, javascript app files and compiled webpack bundles will go in your rails app,
+but all these options are configurable from `config/webpacker.yml`
```

Comment:
Let's split this into two and move it to config/webpack/paths.yml and config/webpack/dev_server.yml.